### PR TITLE
have travis fail when verify-apimachinery.sh fails and fix incompatible apimachinery

### PR DESCRIPTION
**Pull request by chrisz100**
Regarding #5476

### CODE REVIEW

File: pkg/apis/kops/v1alpha1/zz_generated.deepcopy.go [hunk 1]
```diff
@@ -128,6 +128,15 @@ func (in *Assets) DeepCopyInto(out *Assets) {
 			**out = **in
 		}
 	}
+	if in.ContainerProxy != nil {
```
Comment (mikesplain) [inline pkg/apis/kops/v1alpha1/zz_generated.deepcopy.go]: Why is this included here? Doesn't seem related to failing the test

Comment (chrisz100) [reply pkg/apis/kops/v1alpha1/zz_generated.deepcopy.go]: It’s the fix to have tests not fail anymore. But could be split out to another pull request as well.

Comment (mikesplain) [reply pkg/apis/kops/v1alpha1/zz_generated.deepcopy.go]: Discussed in chat. No worries!

**Comment by mikesplain** [comment]
This fixes an apimachery mishap in master since https://github.com/kubernetes/kops/pull/5390

**Comment by justinsb** [comment]
Thanks for fixing @chrisz100  

/approve
/lgtm

**Comment by justinsb** [comment]
/retest

VPC limits

**Comment by mikesplain** [comment]
vpc limit

/retest

**Comment by mikesplain** [comment]
Fixes https://github.com/kubernetes/kops/issues/5476 (for autoclose)

**Pull request merged**